server/src/test/java/org/elasticsearch/index/mapper/CustomSyntheticSourceFieldLookupTests.java

```diff
@@ -0,0 +1,4 @@
+package org.elasticsearch.index.mapper;
+
+public class CustomSyntheticSourceFieldLookupTests {
```

Comment:
todo?

Reply:
Yes, this is in progress. I will undraft the PR once it is ready.